portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,13 +26,22 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  ArrayList<String> data = new ArrayList<>();
```

Comment:
Let's give the variable a more specific name
```suggestion
  ArrayList<String> comments = new ArrayList<>();
```